pyproject.toml

```diff
@@ -101,6 +101,85 @@ build = [
     "setuptools==69.5.1",
     "build==1.2.1"
 ]
+testing = [
+    "deptry>=0.14"
+]
```

Comment:
Right, I will move this to back `dev`. 

Otherwise, we should move the `pytest-*` modules to `testing` as well, and maybe introduce a new dependency class such as `typing` and have `types-*` modules there.